unified-runtime/test/conformance/device/urDeviceGetInfo.cpp

```diff
@@ -2561,6 +2561,9 @@ TEST_P(urDeviceGetInfoTest, SuccessUseNativeAssert) {
 }
 
 TEST_P(urDeviceGetInfoTest, SuccessThrottleReasons) {
+  // TODO: enable when driver/library version mismatch is fixed in CI.
```

Comment:
we like to have tickets accompanying `KNOWN_FAILURE`s, if there is one for this could you link it in the comment

Reply:
Done.